packages/design-picker/src/components/pattern-assembler-cta/index.tsx

```diff
@@ -1,23 +1,19 @@
 import { Button } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
-import type { FC } from 'react';
+import blankCanvasImage from 'calypso/assets/images/onboarding/blank-canvas-cta.svg'; // eslint-disable-line no-restricted-imports
```

Comment:
I thought we should not disable the lint as the reason is to avoid circular deps. Instead, we could put the image into `static/images/onboarding/` folder.

See https://github.com/Automattic/wp-calypso/blob/4033371065bfc50a48909acca50d64e95340a1e3/.eslintrc.js#L56

Reply:
Using `static/images/` folder is discouraged in their [docs](https://github.com/Automattic/wp-calypso/tree/trunk/static/images). The docs in `client/assets/images/` says:

> Technically speaking, webpack will copy that image to the public/images output folder automatically, with an immutable hashed name (so this name will change if the source image changes).

I don't see the risk for any circular dep. Maybe I don't understand it well. Can you help me understand it, please?

Reply:
As the design picker is a package instead of a part of the client code, we have to avoid importing any code from the client folder into packages as much as possible. Even though it's an image and it won't cause any circular dep, I'd still not prefer to do that.

> Using static/images/ folder is discouraged in their [docs](https://github.com/Automattic/wp-calypso/tree/trunk/static/images)

Oh, I didn't notice that. Hmmm...maybe we could create `assets` folder in the `design-picker` package and put the image there. What do you think?

Reply:
Sounds good 👍 Thanks for explaining it